lib/form/html.js

```diff
@@ -137,6 +137,10 @@ const addRequired = (html, input, index) => {
       // add required to input
       const reName = new RegExp(`name=['"]\\s*${inp.name}\\s*['"]`);
       render = render.replace(reName, `name="${inp.name}" aria-required="true" required`);
+
+      // Add required text to label
+      const reLabel = new RegExp('</label>');
+      render = render.replace(reLabel, `<mark class="mark-${input.required}">required to ${input.required}</mark></label>`);
```

Comment:
@ineedsubstance this is coming along great.
Now, you've got to account for both `inp.required` and `input.required`.
You could add a `required` variable up in the `if (inp.required !== undefined) {` conditional (line 93) to capture whether this is publish or save, then use that here.